.drone.yml

```diff
@@ -109,6 +109,8 @@ steps:
     environment:
       GOPROXY: off
       TAGS: bindata sqlite sqlite_unlock_notify
+      GITLAB_READ_TOKEN:
+        from_secret: gitlab_read_token
```

Comment:
if we dont include the secret in the test - lokal unit-tests will always fail (then you have to exclude this test on normal `make test`)

Reply:
gitlab_read_token is already set for PR

Reply:
but it will fail if e.g **I** test gitea localy

Reply:
If I look at last commit it failed at installing git-lfs (must have occurred during the update of the package).

Reply:
I've pulled and merged the latest changes from master. 

The tests are skipped if the token isn't found.